test/test_distributions.py

```diff
@@ -425,6 +425,16 @@ def is_all_nan(tensor):
             'high': torch.tensor([2.0, 3.0], requires_grad=True),
         },
     ]),
+    Example(VonMises, [
+        {
+            'loc': torch.tensor(1.0, requires_grad=True),
+            'concentration': torch.tensor(10.0, requires_grad=True)
+        },
+        {
+            'loc': torch.tensor([0.0, math.pi / 2], requires_grad=True),
+            'concentration': torch.tensor([1.0, 10.0], requires_grad=True)
+        }
+    ]),
```

Comment:
One thing that helps to avoid test seed churn is adding new test examples to the end of  the list, rather than in alphabetic order. Could you try moving both test examples down in this PR to see if test  failures are eliminated?